playbooks/roles/xqueue/defaults/main.yml

```diff
@@ -114,3 +114,5 @@ xqueue_debian_pkgs:
   - libgeos-dev
   # Needed to be able to create the xqueue mysqldb.
   - python-mysqldb
+
+nginx_template_directory: "edx/app/nginx/sites-available"
```

Comment:
This is the same value set in `./playbooks/roles/nginx/defaults/main.yml`, so I suspect we may be able to drop this line entirely.